arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/Hub.java

```diff
@@ -251,6 +254,7 @@ public Hub(final Address l2l1ContractAddress, final Bytes l2l1Topic) {
     this.logData = new LogData(rlpTxrcpt);
     this.logInfo = new LogInfo(rlpTxrcpt);
     this.ecData = new EcData(this, this.wcp, this.ext);
+    this.oob = new Oob(this, (Add) this.add, this.mod, this.wcp);
```

Comment:
Because `add` is declared as  `private final Module add = new Add(this);`. I am not sure why it was declared this way, I guess to write more generic code somewhere.